app/views/layouts/internal.html.erb

```diff
@@ -85,6 +85,8 @@
   .btn {
     min-width: 100px;
     width: fit-content;
+    width: -moz-fit-content;
+    width: -webkit-fit-content;
```

Comment:
I think `moz-fit-content` and `webkit-fit-content` are not needed, since `fit-content` is supported by all browsers except IE and Edge: https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content - see also caniuse: https://caniuse.com/#feat=mdn-css_properties_width_fit-content